app/lib/core/view_models/active_screen_view_model.dart

```diff
@@ -44,4 +46,8 @@ class ActiveScreenViewModel extends BaseViewModel {
     _animalTransportPreviews.clear();
     notifyListeners();
   }
+
+  void navigateToActiveScreen() {
+    _navigationService.pop();
```

Comment:
This should be `_navigationService.popUntil(ActiveScreen.route)` but we can't do that until #162.